app/api/swagger/swaggerconfig.js

```diff
@@ -11,7 +13,6 @@ export default (app) => {
       description: `Uwazi is an open-source solution for building and sharing document collections.
       <br>Remember that using the "Try it out" functionality will execute the requests over your local instalation!`
```

Comment:
This suggests that "trying out the functionality" will execute that in localhost:3000 local environment.

Does removing the host key mean this will now get executed remotely?  Do we really want that change?